### ISSUE TITLE
Eval fails when a prompt string ends with a period "."

### ISSUE THREAD

**Issue by lacom** [promptfoo/promptfoo#1081]
Version 0.68.2

Take the example promptfooconfig.yaml that is generated by the `init` command and modify the prompts by adding a "." at the end like this:

```
# Learn more about building a configuration: https://promptfoo.dev/docs/configuration/guide
description: "My eval"

prompts:
  - "Write a tweet about {{topic}}."
  - "Write a concise, funny tweet about {{topic}}."
  

providers:
  - "openai:gpt-3.5-turbo"
  - "openai:gpt-4o"
  

tests:
  - vars:
      topic: bananas

  - vars:
      topic: avocado toast
    assert:
      # For more information on assertions, see https://promptfoo.dev/docs/configuration/expected-outputs

      # Make sure output contains the word "avocado"
      - type: icontains
        value: avocado

      # Prefer shorter outputs
      - type: javascript
        value: 1 / (output.length + 1)

  - vars:
      topic: new york city
    assert:
      # For more information on model-graded evals, see https://promptfoo.dev/docs/configuration/expected-outputs/model-graded
      - type: llm-rubric
        value: ensure that the output is funny
```

Running `eval` will fail with error "There are no prompts in "Write a tweet about {{topic}}."

**Comment by typpo** [comment]
Thanks for the easy repro.  Fix in #1084

**Comment by mldangelo** [comment]
@lacom a fix for this was deployed in [0.68.3](https://github.com/promptfoo/promptfoo/releases/tag/0.68.3) Please let us know if you run into any other issues and thanks again for reporting it!

**Comment by lacom** [comment]
@mldangelo 👊 awesome! Thanks for the fix!
